includes/class-wc-siftscience-admin.php

```diff
@@ -158,14 +158,25 @@ private function output_settings_stats() {
 			}
 
 			$stats = get_option( WC_SiftScience_Options::STATS, 'none' );
-			if ( 'none' === $stats ) {
+			if ( $stats === 'none' ) {
 				echo '<p>No stats stored yet</p>';
 				return;
 			}
 
-			$stats = json_decode( $stats );
-			$stats = json_encode( $stats, JSON_PRETTY_PRINT );
-			echo '<pre>' . $stats . '</pre>';
+			$stats = json_decode( $stats , true );
+
+			foreach ( array_keys( $stats ) as $outer_v ) {
```

Comment:
Even better: `foreach ( $stats as $key => $value )`.

Reply:
I'm iterating keys i have no need for value (array structure )

Reply:
Your are using the value a little late

Reply:
???